app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
Missing annotation here. The 'Cafe' class is imported but unused.
```suggestion
def go_to_cafe(friends: list[str], cafe: Cafe) -> str:
```